### PR TITLE
Speed Buff to rogue + small tweaks

### PR DESCRIPTION
**Pull request by OrbitalParadise**
DISCUSS

### CODE REVIEW

File: code/modules/jobs/job_types/roguetown/adventurer/types/combat/rogue.dm [hunk 1]
```diff
@@ -31,15 +31,15 @@
...
 	H.mind.adjust_skillrank(/datum/skill/combat/unarmed, 3, TRUE)
-	H.mind.adjust_skillrank(/datum/skill/combat/knives, 3, TRUE)
+	H.mind.adjust_skillrank(/datum/skill/combat/knives, 4, TRUE)
 	H.mind.adjust_skillrank(/datum/skill/combat/polearms, 1, TRUE)
 	H.mind.adjust_skillrank(/datum/skill/misc/swimming, 3, TRUE)
-	H.mind.adjust_skillrank(/datum/skill/misc/climbing, rand(4,6), TRUE)
+	H.mind.adjust_skillrank(/datum/skill/misc/climbing, pick(5,6), TRUE)
 	H.mind.adjust_skillrank(/datum/skill/craft/crafting, 1, TRUE)
-	H.mind.adjust_skillrank(/datum/skill/misc/reading, pick(0,1,1), TRUE)
+	H.mind.adjust_skillrank(/datum/skill/misc/reading, 1, TRUE)
```
Comment (Real-MAGNUM) [inline code/modules/jobs/job_types/roguetown/adventurer/types/combat/rogue.dm:39]:
```suggestion
	H.mind.adjust_skillrank(/datum/skill/misc/reading, pick(0,1,1), TRUE)
```

File: code/modules/jobs/job_types/roguetown/adventurer/types/combat/rogue.dm [hunk 2]
```diff
@@ -58,5 +58,5 @@
 	ADD_TRAIT(H, TRAIT_STEELHEARTED, TRAIT_GENERIC)
 	H.change_stat("strength", -1)
 	H.change_stat("perception", 2)
-	H.change_stat("speed", 2)
+	H.change_stat("speed", 4)
```
Comment (Real-MAGNUM) [inline code/modules/jobs/job_types/roguetown/adventurer/types/combat/rogue.dm:61]:
```suggestion
	H.change_stat("speed", 3)
```

Comment (Real-MAGNUM) [reply code/modules/jobs/job_types/roguetown/adventurer/types/combat/rogue.dm]: made an edit to include 3/4 at a 50%

File: code/modules/jobs/job_types/roguetown/adventurer/types/combat/rogue.dm (hunk 1 above)
Comment (OrbitalParadise) [reply code/modules/jobs/job_types/roguetown/adventurer/types/combat/rogue.dm:39]: 2 int but a 33% chance to have 0 in reading?

**Comment by Real-MAGNUM** [comment]
I'm fine with rogues being a quick dagger focused class. They could use the speed now that crossbows have to be ordered from a smith or stolen from guards.

**Review by Real-MAGNUM — APPROVED** [review]
Approved

**Pull request merged**